README.md

```diff
@@ -1,15 +1,41 @@
 # Linuxbrew Extra
 
+[Linuxbrew Extra](https://github.com/Linuxbrew/homebrew-extra) is a collection of
+formulae for [Linuxbrew](https://github.com/Linuxbrew/brew) that are not dependencies of
+[Linuxbrew core formulae](https://github.com/Linuxbrew/homebrew-core) and are specific to Linux.
+
 ## How do I install these formulae?
-`brew install linuxbrew/extra/<formula>`
 
-Or `brew tap linuxbrew/extra` and then `brew install <formula>`.
+Assuming that you have [Linuxbrew/brew](https://github.com/Linuxbrew/brew) installed, execute:
 
-Or install via URL (which will not receive updates):
+### Method 1
+```
+brew tap linuxbrew/extra # needed only once
+brew install <formula>   # for example:  brew install man-db
+```
+
+### Method 2
+You can also install your first formula in one step:
+```
+brew install linuxbrew/extra/<formula>
+```
+This will automatically add this repository to the list of repositories in your Linuxbrew installation (it will _tap_ this repository) and later on you won't have to specify `linuxbrew/extra` prefix when installing Linuxbrew/extra formulae.
+
+### Method 3
+
+Just like any other formula, you can install any Linuxbrew/extra formula _via_ its URL:
```

Comment:
Simplicity is better here I think. Replace with…
```
`brew install linuxbrew/extra/FORMULA` or `brew tap linuxbrew/extra` and then `brew install FORMULA`.
```
and remove `Or install via URL`